hack/run-ci-e2e-test.sh

```diff
@@ -133,10 +133,15 @@ fi
 
 echo "Testing against Windows Server $WIN_VER"
 
+if [ -z "$MIRROR_REGISTRY_HOST" ]; then
+    echo "env MIRROR_REGISTRY_HOST not found"
+    return 1
+fi
```

Comment:
Is this needed? `MIRROR_REGISTRY_HOST` will only be set in the disconnected job

Reply:
thats true I wanted to fail early for CI testing, will remove this soon